telemetry/spans.go

```diff
@@ -121,7 +122,8 @@ func (c *spanCommonBlock) Bytes() []byte {
 
 // SpanBatch represents a single batch of spans to report to New Relic.
 type SpanBatch struct {
-	Spans []Span
+	Common *SpanCommonBlock
```

Comment:
We may want to consider keeping the span common block out of the span batch. The initial design treats the common block as a separate PayloadEntry, and the RequestFactory knows how to combine the different PayloadEntrys together.

Reply:
That's still true here. Multiple span batches can be combined in a single request body like so:
```
[
  { // Span batch 1
    "common: {"attributes: {"foo":"bar"}},
    "spans": [{..},{...}]
  },
  { // Span batch 2
    "common: {"attributes: {"baz":"qux"}},
    "spans": [{..},{...}]
  }
]
```